src/components/_cards/PortfolioCard/index.tsx

```diff
@@ -190,9 +189,6 @@ export const PortfolioCard: VFC<BoxProps> = (props) => {
                         : "--")}
                   </CardStat>
                 </VStack>
-                {isMounted && (
```

Comment:
let's don't remove this, just comment this out and add a comment why we disable it